enterprise/frontend/src/metabase-enterprise/sandboxes/containers/QuestionParameterTargetWidget.tsx

```diff
@@ -0,0 +1,69 @@
+import { useDisclosure } from "@mantine/hooks";
+
+import { QuestionLoaderHOC } from "metabase/common/components/QuestionLoader";
+import SelectButton from "metabase/common/components/SelectButton";
+import { ParameterTargetList } from "metabase/parameters/components/ParameterTargetList";
+import {
+  getMappingOptionByTarget,
+  getParameterMappingOptions,
+} from "metabase/parameters/utils/mapping-options";
+import { Popover } from "metabase/ui";
+import type Question from "metabase-lib/v1/Question";
+import type { ParameterTarget } from "metabase-types/api";
+
+type QuestionParameterTargetWidgetProps = {
+  question: Question;
+  target?: ParameterTarget | null;
```

Comment:
I'm not sure this is the right type, it doesn't match the usage in DataAttributeMappingEditor, at least